chromium_src/chrome/browser/profiles/profile_io_data.cc

```diff
@@ -15,6 +15,8 @@
 bool ProfileIOData::IsHandledProtocol(const std::string& scheme) {
   if (scheme == kBraveUIScheme)
     return true;
+  if (scheme == "ipfs")
```

Comment:
maybe kIPFSScheme? and how about kIPNSScheme?

Reply:
good call, I forgot to update it after testing if it was the right change 👍

Reply:
I don't think I can use kIPFSScheme and kIPNSScheme here easily without patching in a dep to chrome/browser to components/ipfs. Maybe best to just keep it a string comparison?

Reply:
lgtm